.npmignore

```diff
@@ -1 +0,0 @@
-bench
```

Comment:
@feross are `.npmignore` better for you than `files`? I always thought that white list is better.